### PR TITLE
Update C++ SDK version to 7.2.0, and dependencies to M93.

### PR DESCRIPTION
**Pull request by jonsimantov**

### CODE REVIEW

**Comment by jonsimantov** [comment]
(Working on updating the readme file, stand by for that.)

File: Android/firebase_dependencies.gradle [hunk 1]
```diff
@@ -16,30 +16,30 @@ import org.gradle.util.ConfigureUtil;
...
 // A map of library to the dependencies that need to be added for it.
 def firebaseDependenciesMap = [
-  'app' : ['com.google.firebase:firebase-analytics:18.0.2'],
-  'admob' : ['com.google.firebase:firebase-ads:19.7.0',
-             'com.google.android.gms:play-services-measurement-sdk-api:18.0.2',
+  'app' : ['com.google.firebase:firebase-analytics:18.0.3'],
+  'admob' : ['com.google.firebase:firebase-ads:20.0.0',
+             'com.google.android.gms:play-services-measurement-sdk-api:18.0.3',
              'com.google.android.gms:play-services-base:17.6.0'],
   'analytics' : ['com.google.firebase:firebase-analytics:18.0.2',
```
Comment (a-maurice) [inline Android/firebase_dependencies.gradle:23]: This line didn't get updated correctly

File: release_build_files/Android/firebase_dependencies.gradle [hunk 2]
```diff
@@ -16,29 +16,29 @@ import org.gradle.util.ConfigureUtil;
...
 // A map of library to the dependencies that need to be added for it.
 def firebaseDependenciesMap = [
-  'app' : ['com.google.firebase:firebase-analytics:18.0.2'],
-  'admob' : ['com.google.firebase:firebase-ads:19.7.0',
-             'com.google.android.gms:play-services-measurement-sdk-api:18.0.2',
+  'app' : ['com.google.firebase:firebase-analytics:18.0.3'],
+  'admob' : ['com.google.firebase:firebase-ads:20.0.0',
+             'com.google.android.gms:play-services-measurement-sdk-api:18.0.3',
              'com.google.android.gms:play-services-base:17.6.0'],
   'analytics' : ['com.google.firebase:firebase-analytics:18.0.2',
```
Comment (a-maurice) [inline release_build_files/Android/firebase_dependencies.gradle:23]: Same here

**Review by a-maurice — CHANGES_REQUESTED** [review]
Also need to update cpp_version_sdk.json

**Comment by jonsimantov** [comment]
> Also need to update cpp_version_sdk.json

Done.

**Review by a-maurice — DISMISSED** [review]

**Review by a-maurice — APPROVED** [review]

**Pull request merged**